DesktopUI2/DesktopUI2/ViewModels/HomeViewModel.cs

```diff
@@ -535,7 +535,7 @@ private void GenerateMenuItems()
         new MenuItemViewModel(LaunchManagerCommand, "Manage accounts in Manager", MaterialIconKind.AccountCog)
       );
 
-      menu.Items.Add(new MenuItemViewModel(RefreshCommand, "Refresh streams & accounts", MaterialIconKind.Refresh));
+      menu.Items.Add(new MenuItemViewModel(RefreshCommand, $"Refresh projects & accounts", MaterialIconKind.Refresh));
```

Comment:
A few of these strings have an unnecessary $
Perhaps we can quickly remove them before his pr is merged